### ISSUE TITLE
Panel : The color contrast of keyboard focus for several controls is less than 3:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#99426]
**Environment Details:**
VSCode Version : Environment Details:
VSCode Version : 1.41.0-insider
OS Version : MAC 10.15.5

**Additional Details:**
Screenreader: Voiceover
GitHubTags: #A11y_VisualStudioCode;#CAITestPass_May2020;#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11y_VSCode;#MAS1.4.11;#A11ySev2;#FTP;#BM

**Steps to Reproduce:**
1.Open Visual studio code insider editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3.Using "Ctrl+" key to split the terminal
4.Navigate to Spilt terminal icon using tab and turn on color contrast analyzer.

**Actual:**
When low vision users navigate to the controls using keyboard focus to several controls present on the screen will not be able to view the keyboard focus of the control as it has a color contrast ratio of less than 3:1.

**Expected:**
The color contrast ratio of the keyboard focus should be equal to or more than 3:1 for the controls on the screen.

**Attachment For Reference:**
<img width="1682" alt="Screenshot 2020-06-05 at 12 27 14 PM" src="https://user-images.githubusercontent.com/52036772/83846372-1d47da00-a728-11ea-9e89-3155b42d115d.png">

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by misolori** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/99244

**Comment by v-umnaik** [comment]
A11yMAS;Visual Studio Code;CAITestPass_May2020;Win10;A11yTCS;A11Y_Visual Studio Code_Win32(Benchmark May 2020)
